packages/networks/src/networks/vNaga/shared/helpers/createRequestId.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Creates a unique request ID for tracking requests
+ */
+export function createRequestId(): string {
+  return `request_${Date.now()}_${Math.random().toString(36).substring(2, 15)}`;
```

Comment:
this is the main request id right?  can we prefix with lit_ instead of request_ for consistency with what we have now?